city_scrapers/spiders/det_regional_convention.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime, time
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class DetRegionalConventionSpider(CityScrapersSpider):
+    name = "det_regional_convention"
+    agency = "Detroit Regional Convention Facility Authority"
+    timezone = "America/Detroit"
+    start_urls = ["http://www.drcfa.org/upcoming-meetings"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+        for item in response.xpath("//div[@class='link']/ul/li/a"):
```

Comment:
It looks like they have posted details higher up on the page, it could be worth testing it by parsing this archived page so that it'll work if they post again https://web.archive.org/web/20200213121350/http://www.drcfa.org/upcoming-meetings

Reply:
should I add the archived page instead of the actual page in spider tests?

Reply:
I think that makes sense! We should still use the URL you have now in `start_urls`, but you can change it to the archive one for testing